keras_rcnn/heads/resnet.py

```diff
@@ -0,0 +1,96 @@
+import keras
+
+
+def ResHead(classes, mask=False):
+    """Resnet heads as in Mask R-CNN."""
+    def f(x):
+        if keras.backend.image_data_format() == "channels_last":
+            channel_axis = 3
+        else:
+            channel_axis = 1
+
+        y = keras.layers.TimeDistributed(
```

Comment:
@0x00b1 I agree with you here. But I can hardly see how to implement such a wrapper. 
For example, we have `ResHead` that takes feature `x` as input and returns some outputs `y`. We need to apply TimeDistributed to each of the layers in between.
Maybe you have good ideas how to do that?